src/backend/core/models.py

```
@@ -488,6 +494,56 @@ class Meta:
+    def adresse_messagerie_domain(self):
+        """Get the mail domain for the organization."""
+        if not self.adresse_messagerie:
+            return None
+        return self.adresse_messagerie.split("@")[1]
+
+
+    def site_internet_domain(self):
+        """Get the website domain for the organization."""
+        if not self.site_internet:
+            return None
+        print(self.site_internet)
```

Ruff (0.14.7)

`print` found

Remove `print`

(T201)